gulpfile.js

```diff
@@ -0,0 +1,57 @@
+'use strict';
+
+var gulp = require('gulp');
+var path = require('path');
+var del = require('del');
+var source = require('vinyl-source-stream');
+var $ = require('gulp-load-plugins')();
+
+var assetPath = 'vendor/assets/javascripts/rolodex_angular';
+var distPath = 'dist';
+var tmpPath = '.tmp';
+
+// Compiles coffee to js, annotates Angular dependencies
+gulp.task('js', ['js-hack'], function() {
+  return gulp.src(path.join(assetPath, '/**/*.coffee'))
+    .pipe($.coffee())
+    .pipe($.ngAnnotate())
+    .pipe(gulp.dest(tmpPath))
+});
+
+// A hack for non-Ruby projects to fake the templates namespace (which lives in a gem)
+gulp.task('js-hack', function() {
+  var stream = source('templates.js');
+  stream.end("(function() { angular.module('templates', []); }).call(this);");
```

Comment:
I wouldn't encourage calling this a hack as a name for a task. How about `js-template-cache` or some such